test/resources/fixtures/test-statements.js

```diff
@@ -83,6 +83,12 @@ module.exports = {
     envelope: [ SIMPLE_STMT_1.publisher ],
     envelopeEmpty: [ ENVELOPE_STMT.publisher ]
   },
+  expectedDeps: {
+    simple: [ new Set(['dep1', 'dep2']), new Set(['dep1', 'dep3']) ],
+    compound: [ new Set() ],
```

Comment:
we should test that it retrieves dependencies correctly in compound, rather than envelopes; we don't have any of the latter yet.